src/EfficientNetCheckPointFactory.ts

```diff
@@ -6,11 +6,22 @@ const defaultModelsUrl =
 const modelFileName = "model.json";
 const inputLayerImageSize = [224, 240, 260, 300, 380, 456, 528, 600];
 
+interface EfficientNetCheckPointFactoryOptions {
+  localModelRootDirectory?: string;
+}
+
 export default class EfficientNetCheckPointFactory {
   static async create(
-    checkPoint: EfficientNetCheckPoint
+    checkPoint: EfficientNetCheckPoint,
+    options?: EfficientNetCheckPointFactoryOptions
   ): Promise<EfficientNetModel> {
-    const modelPath = `${defaultModelsUrl}${checkPoint}/${modelFileName}`;
+    const { localModelRootDirectory } = options || {};
+    let modelPath = `${defaultModelsUrl}${checkPoint}/${modelFileName}`;
+
+    if (localModelRootDirectory) {
```

Comment:
this want work please follow this documentation on tensorflowJs

https://js.tensorflow.org/api/latest/#loadGraphModel